eng/Subsets.props

```diff
@@ -499,10 +499,6 @@
 
   <Choose>
     <When Condition="$(_subset.Contains('+packs.product+'))">
-      <ItemGroup Condition="'$(BuildNativeAOTRuntimePack)' != 'true' and '$(RuntimeFlavor)' != 'Mono'">
-        <SharedFrameworkProjectToBuild Include="$(InstallerProjectRoot)pkg\sfx\Microsoft.NETCore.App\Microsoft.NETCore.App.Runtime.Composite.sfxproj" />
```

Comment:
Delete these projects too? Or do we expect them to be still used in some situations?

Reply:
Wanted to leave in place, since there is a bunch of plumbing collateral. @ivdiazsa if the asp.net projects use similar mechanisms we can certainly remove from runtime rep.

Reply:
ok will delete it then, should be available in other branches if we need to refer to them.